docs/encoder.md

```diff
@@ -3,6 +3,8 @@ Add twist control to your keyboard! Volume, zoom, anything you want
 
 I2C encoder type has been tested with the Adafruit I2C QT Rotary Encoder with NeoPixel
 
+**Note:** If using a split keyboard best to use the [encoder-scanner](scanners.md) encoder explained at the bottom of scanners docs.
```

Comment:
```suggestion
**Note:** If using a split keyboard best to use the encoder-scanner explained at the bottom of [scanners docs](scanners.md).
```
Link target and name should have semantic proximity.

Here you could be a bit more verbose: If split with encoders and both sides should work, it's currently necessary to ...